packages/grafana-ui/src/components/TextArea/TextArea.story.tsx

```diff
@@ -12,36 +12,27 @@ export default {
     docs: {
       page: mdx,
     },
-  },
-};
-
-export const simple = () => {
-  const BEHAVIOUR_GROUP = 'Behaviour props';
-  // ---
-  const invalid = boolean('Invalid', false, BEHAVIOUR_GROUP);
-  const disabled = boolean('Disabled', false, BEHAVIOUR_GROUP);
-
-  const VISUAL_GROUP = 'Visual options';
-  // ---
-  const placeholder = text('Placeholder', 'This is just a placeholder', VISUAL_GROUP);
-  const cols = number('Cols', 30, { range: true, min: 5, max: 50, step: 5 }, VISUAL_GROUP);
-  const CONTAINER_GROUP = 'Container options';
-  // ---
-  const containerWidth = number(
-    'Container width',
-    300,
-    {
-      range: true,
-      min: 100,
-      max: 500,
-      step: 10,
+    knobs: {
+      disable: true,
     },
-    CONTAINER_GROUP
-  );
+  },
+  argTypes: {
+    cols: { control: { type: 'range', min: 5, max: 50, step: 5 } },
+    containerWidth: { control: { type: 'range', min: 100, max: 500, step: 10 } },
+  },
+} as Meta;
 
+export const Simple: Story = (args) => {
```

Comment:
Maybe we should follow the convention and call the first story `Basic`?
```suggestion
export const Basic: Story = (args) => {
```

Reply:
Alright. 👍